app/views/PDFView.scala.html

```diff
@@ -74,7 +76,7 @@ <h3 class="govuk-heading-s govuk-!-static-margin-bottom-1">@messages("pdf.utr")<
                         <div class="about-results">
                             <h3 class="govuk-heading-s about-results-border">@messages("pdf.aboutThisResult")</h3>
                             <h4 class="govuk-heading-xs">@messages("pdf.dataOfResult")</h4>
-                            <p class="govuk-body about-results-border">@Html(LocalDateTime.now().format(DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm:ss")).toString())</p>
+                            <p class="govuk-body about-results-border">@Html(ZonedDateTime.of(LocalDateTime.now(), ZoneId.of("UTC")).format(DateTimeFormatter.ofPattern("dd MMMM yyyy HH:mm:ss z")).toString())</p>
```

Comment:
Can we move this to a helper? DateUtils?

Call it formatZonedDateTime?

Reply:
Thats updated